flexget/plugins/filter/exists_series.py

```diff
@@ -33,7 +33,9 @@ class FilterExistsSeries(object):
                 'type': 'object',
                 'properties': {
                     'path': one_or_more({'type': 'string', 'format': 'path'}),
-                    'allow_different_qualities': {'enum': ['better', True, False], 'default': False}
+                    'allow_different_qualities': {'enum': ['better', True, False], 'default': False},
+                    'date_dayfirst': {'type': 'boolean', 'default': False},
```

Comment:
Not sure if this PR is still relevant, but setting these defaults is a bad idea. The underlying code in the parsers will try different pairs of `True` and `False` if these options are _not_ set.